SECURITY.md

```diff
@@ -12,3 +12,9 @@ although we keep your name confidential if you request it.
 
 Reported vulnerabilities will be analyzed swiftly.
 
+## OWASP dependency checks
+
+Dependencies of this project is checked against known vulnerabilities with *OWASP dependency-check* -tool.
```

Comment:
Grammar nit-picks:
Dependencies ... **are** checked against ... with **the** ...
... once **per** day ...
... can be found **(no from)** here ....